rbc/omnisci_backend/__init__.py

```diff
@@ -1,13 +1,16 @@
 from .numpy_funcs import *  # noqa: F401, F403
-from .npyimpl import *  # noqa: F401, F403
-from .mathimpl import *  # noqa: F401, F403
+from .numpy_ufuncs import *  # noqa: F401, F403
 from .omnisci_array import *  # noqa: F401, F403
 from .omnisci_column import *  # noqa: F401, F403
 from .omnisci_bytes import *  # noqa: F401, F403
 from .omnisci_metatype import *  # noqa: F401, F403
 from .omnisci_text_encoding import *  # noqa: F401, F403
-from .numpy_ufuncs import *  # noqa: F401, F403
 from .omnisci_pipeline import *  # noqa: F401, F403
-from .python_operators import *  # noqa: F401, F403
 from .omnisci_column_list import *  # noqa: F401, F403
 from .omnisci_table_function_manager import *  # noqa: F401, F403
+
+import rbc.omnisci_backend.mathimpl as math  # noqa: F401
```

Comment:
I think you can revert this one. Users don't use math from `rbc.omnisci_backend.mathimpl` but rather importing math directly:

```python
import math

@omnisci('float64(float64)')
def log2(x):
    return math.log2(x)
```

The same for python_operators module.

Reply:
It would still work but I did this so that we don't pollute the env with what's inside these when we do `import rbc.omnisci_backend`.